lfs/gitscanner.go

```diff
@@ -140,6 +140,20 @@ func (s *GitScanner) ScanRefRange(left, right string, cb GitScannerFoundPointer)
 	return scanLeftRightToChan(s, callback, left, right, s.cfg.GitEnv(), s.cfg.OSEnv(), opts)
 }
 
+// ScanRefRangeByTree scans through all commits from the given left and right
+// refs.
```

Comment:
I wondered about this comment -- the one for `ScanRefByTree()` below says "scans through all trees in the current ref", while here it's "all commits".  Should this comment also reflect the "all trees" version of the text?

Reply:
Updated.